### ISSUE TITLE
[Feature]: Add the ability to calculate indices in terms of numbers

### ISSUE THREAD

**Issue by ChristineStawitz-NOAA** [NOAA-FIMS/FIMS#315]
### Is your feature request related to a problem? Please describe.

Many indices of abundance are calculated in numbers, but FIMS only calculates indices in biomass

### Describe the solution you would like.

Add CalculateAbundanceIndex() to population and a vector to store the abundance index as part of the fleet data

### Describe alternatives you have considered

Right now we only have biomass indices, so users would have to externally convert

### Statistical validity, if applicable

It's generally considered good statistical practice to fit to the data that are collected

### Describe if this is needed for a management application

Many states provide data as numbers

**Comment by kellijohnson-NOAA** [comment]
I moved this to the Parking Lot because there is a workaround for the case studies, i.e., translate numbers to biomass before creating the data. We can revisit its status on Tuesday when we talk about case-study prioritization.

**Comment by kellijohnson-NOAA** [comment]
@nathanvaughan-NOAA you are working on this now, right?

**Comment by nathanvaughan-NOAA** [comment]
Yes, you will be able to model indices and landings in terms of numbers or
weight.

*Nathan R. Vaughan Ph.D.*

Research Scientist, Contractor/Affiliate with Vaughan Analytics in support
of
NOAA Fisheries - Southeast Fisheries Science Center |  U.S. Department of
Commerce

Mobile: (305) 733-6643

www.fisheries.noaa.gov

**Comment by kellijohnson-NOAA** [comment]
Closed with c83e2bacc29de6c8fa1e1aa0981f158108a9bfb5
